test/specs/upgrade/queryservice-post-upgrade.ts

```diff
@@ -93,14 +92,14 @@ describe( 'Wikibase post upgrade', function () {
 			bindings
 		);
 
-		assert( dateModified !== null );
-		assert( schemaVersion !== null );
-		assert( siteLinks !== null );
-		assert( identifiers !== null );
-		assert( timestamp !== null );
-		assert( statement !== null );
+		expect( dateModified ).toEqual( expect.anything() );
+		expect( schemaVersion ).toEqual( expect.anything() );
+		expect( siteLinks ).toEqual( expect.anything() );
+		expect( identifiers ).toEqual( expect.anything() );
+		expect( timestamp ).toEqual( expect.anything() );
+		expect( statement ).toEqual( expect.anything() );
```

Comment:
which expect are we using? I cannot find `anything()` in [wdio doc](https://webdriver.io/docs/api/expect-webdriverio) but in [jest doc](https://jestjs.io/docs/expect)

Reply:
You get Jest's matchers and testers for free with wdio's expect: https://webdriver.io/docs/api/expect-webdriverio/#default-matchers